src/components/login.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import { getUsers } from "../utils/api";
+import { useLogin } from "../contexts/login";
+
+const Login = () => {
+  const [usernameInput, setUsernameInput] = useState("");
+
+  const { user, setUser } = useLogin();
+
+  function handleClick(usernameInput) {
+    console.log("clicked");
```

Comment:
Please _always_ remove unused code before committing. In the workplace this would result in your code being rejected.